src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -34,14 +35,26 @@ static Build()
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
+        
+        public string Commit { get; set; }
 
-        public string Repository { get; set; }
+        public int AzureDevOpsBuildId { get; set; }
 
-        public string Branch { get; set; }
+        public int AzureDevOpsBuildDefinitionId { get; set; }
 
-        public string Commit { get; set; }
+        public string AzureDevOpsAccount { get; set; }
```

Comment:
Can we use Owned Types for the AzDev part of this class? I'd like to organize it that way.

Reply:
Are you fine with using Owned Types just for AzDev? I thought it would be strange to use owned types only for AzDev and leave GitHub info on the Build property. The overall code is also much simpler if not using owned types.